examples/tui/button.js

```
@@ -0,0 +1,203 @@
+
+function isInsideButton(x, y) {
+  return (
+    x >= button.x &&
+    x < button.x + button.width &&
+    y >= button.y &&
+    y < button.y + button.height
+  );
+}
+
+function parseMouseEvent(seq) {
+  const match = seq.match(/^\x1b\[<([0-9]+);([0-9]+);([0-9]+)([mM])$/);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)